packages/material-ui/src/Popper/Popper.js

```diff
@@ -79,7 +79,7 @@ const Popper = React.forwardRef(function Popper(props, ref) {
 
   React.useEffect(() => {
     if (popperRef.current) {
-      popperRef.current.update();
+      popperRef.current.forceUpdate();
```

Comment:
Technically isn't needed since `update()` is still there but the docs say `forceUpdate()` replaced it

Reply:
This is also the better solution. In popper v1 `update` was async and scheduled a microtask. Since React is now in full control of scheduling it's better to defer to React for scheduling. Or rather prefer cooperative scheduling over each package using their own schedule.

Just spent some hours debugging this issue and a sync version of `update` (i.e not debounced via Promise.resolve) is way more predictable.